src/main/java/nl/tudelft/contextproject/script/Script.java

```diff
@@ -152,7 +173,9 @@ public Shot getCurrentShot() {
         try {
             return shots.get(current);
         } catch (Exception e) {
-            return null;
+            Camera dummyCamera = new Camera();
+            dummyCamera.setNumber(-1);
+            return new Shot(-1, "-1", dummyCamera, new InstantPreset(new CameraSettings(), -1), "No shot");
```

Comment:
Using a dummy shot value can be a good idea, but not in this way. Creating a new Camera using `new Camera()` also increases the number of cameras there currently are in the system. So when you have cameras 1, 2 and 3, you go through this method and then add another camera, then it will have id 5 instead of 4. Seeing as this can be fucked up real easily, I'll fix this for you.